src/util/scroll-view.ts

```diff
@@ -68,6 +68,20 @@ export class ScrollView {
     this._eventsEnabled = true;
   }
 
+  setScrolling(isScrolling: boolean, ev: ScrollEvent) {
```

Comment:
review with @adamdbradley (explain use case)
add support for JS-scrolling? maybe not needed

add new unit tests